packages/rancher-windows-exporter/charts/templates/daemonset.yaml

```diff
@@ -66,7 +66,7 @@ spec:
           path: \\.\pipe\rancher_wins_proxy
       - name: binary-host-path
         hostPath:
-          path: c:/etc/windows-exporter
+          path: {{ .Values.global.cattle.rkeWindowsPathPrefix }}etc/windows-exporter
```

Comment:
Looks good, but can you add validation of the rkeWindowsPathPrefix value in this PR too?

e.g. add [`{{ include "winsUpgrader.validatePathPrefix" . }}`](https://github.com/rancher/wins/blob/4bf76f4ecbc0ea7b835a4080d36705737a41559a/charts/rancher-wins-upgrader/templates/configmap.yaml#L1) to the top of this file and add that function to the [`_helpers.tpl`](https://github.com/rancher/wins/blob/4bf76f4ecbc0ea7b835a4080d36705737a41559a/charts/rancher-wins-upgrader/templates/_helpers.tpl#L32-L39).

Reply:
Done